### DISCUSSION TITLE
build libsndfile for Apple macOS with M1

### DISCUSSION THREAD

**Discussion by joelkraehemann** [libsndfile/libsndfile#986] [Q&A] [answered]
Hi,

I don't get any sound out of my WAV files using libsndfile on macos. I compiled it from source and actually there is no other option to me. (Don't even think in suggesting macports or even homebrew) 

I have placed the sse2neon.h in libsndfile's src directory.

https://github.com/DLTcollab/sse2neon

Patched src/config.h.in with:

```
/* Define to 1 if you have the <immintrin.h> header file. */
#define HAVE_IMMINTRIN_H 1
```

modified src/common.h to include sse2neon.h instead of immintrin.h.

I configured the project like following:

```
CC=clang CXX=clang++ CPPFLAGS="-DUSE_SSE2=1" ./configure --prefix=$HOME/gtk/inst --disable-alsa --disable-octave --disable-silent-rules --disable-sqlite --disable-test-coverage
make
make install
```

But still no success. Help is going to be appreciated.

**Comment by joelkraehemann** [comment, marked as answer]
Hi,

Actually it works fine, the problem was more with libinstpatch that uses libsndfile. Obviously, I had a bug related to sample selection.

regards, Joël

**Comment by evpobr** [comment]
Hi @joelkraehemann . I am glad that you have solved your problem.
